openstack_sdk/common.py

```diff
@@ -20,9 +20,15 @@
 import openstack
 
 
+class QuotaException(Exception):
+    pass
+
+
 class OpenstackResource(object):
+    resource_type = None
```

Comment:
What is the purpose of `resource_type`?

Reply:
Well, this will be used in order to get the qouta for each resources like the following:
1. get_compute_quota
2. get_volume_quota

and so on so basically all compute resources will have compute, all network stuff will have network type so that it can be used to fetch the right resources quota